store/badger_test.go

```diff
@@ -0,0 +1,74 @@
+package store
+
+import (
+	"testing"
+
+	"github.com/dgraph-io/badger"
+)
+
+func TestGet(t *testing.T) {
```

Comment:
This tests only error situations:
```suggestion
func TestGetErrors(t *testing.T) {
```